src/main/resources/publishqueue/component.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
com.ibm.mq.allclient.jar, com.ibm.mq.allclient.jar, com.ibm.mq.allclient.jar these jar files should be remove. Please remove all the proprietary IBM binary files from the PR.